src/encoded/docs/schema-changes.md

```diff
@@ -154,44 +154,114 @@ This new object is an array of example objects that can successfully POST agains
+        )
+
+5. Document the changes to the corresponding log file within the **schemas/changelogs** directory. For example, a minor change in the treatment object after version 11 that allowed one to use *μg/kg* as treatment units is shown below:
 
-3. Update sample data, **data/inserts** directory, to test the changes made to the schema in **tests** directory. 
+        ### Minor changes since schema version 11
 
+        * *μg/kg* can now be specified as amount units.
 
 ### Update schema version
+* Schema version has to be updated (bumped up by 1) if the change that is being introduced will lead to a potential invalidation of existing objects in the database. 
+
+* Examples include:
+    1) Changing the name of a property in the existing schema.
+    2) Removing a property from the existing schema.
+    3) A property that previously allowed free text is now changed to a possible list of allowed enums.
+    4) If an existing enum is removed from a property.
+
+
+* Most of the cases described above are examples where there will be a potential conflict for all the existing objects to be validated under the new schema. Hence, an additional step of adding an upgrader script will be required. This will ensure that all the existing objects will be changed such that they can fit into the new schema that is currently being implemented.
+
+* One more case needs a mention here: When adding multiple new schema properties that would lead to substantial changes within an existing schema, one must update the schema version. Even though, the addition of multiple new schema properties is not going to invalidate any existing objects, it would be useful to do so. Particularly, this will be helpful to all the submitters and users who are trying to use these properties either to submit their data or while querying the database using scripts.
+
+**Follow the steps as outlined below**
 
-1. In the **schemas** directory, increment the schema version (1->2), if it meets the above rule: 
+1. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version. One of the possible enums for for the genetic modifications object was "validation" up until schema version 6. As a part of schema version upate to 7, "validation" was removed and instead the term "characterization" was added.
+
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "validation",
+                "screening",
+                "expression"
+            ]
+        },
+        # Changing validation to characterization as a list of enums within the purpose property:
```

Comment:
# Replacing the enum "validation" by the enum "characterization" in the list of enums of the "purpose" property